src/session_info2/_dists.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: MPL-2.0
+from __future__ import annotations
+
+import re
+from importlib.metadata import Distribution, distributions
```

Comment:
Are there docs for this `distributions` function https://docs.python.org/3/library/importlib.metadata.html? Presumably it gets all packages in the current env or?

Reply:
Yup, @jaraco said so: https://github.com/python/importlib_metadata/issues/494#issuecomment-2248273306